spec/plugin/r_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe "R" do
+  let(:filename) { 'test.r' }
+
+  before :each do
+    vim.set(:expandtab)
+    vim.set(:shiftwidth, 2)
+  end
+
+  after :each do
+    vim.command('silent! unlet g:r_indent_align_args')
+  end
+
+  specify "function calls with align_args = 0" do
+    vim.command('let g:r_indent_align_args = 0')
+
+    set_file_contents 'print(1, a = 2, 3)'
+    vim.search('1,')
+    split
+
+    assert_file_contents <<~EOF
+      print(
+        1,
+        a = 2,
+        3
+      )
+    EOF
+
+    join
+
+    assert_file_contents 'print(1, a = 2, 3)'
+  end
```

Comment:
Currently this test is failing. I tried debugging it, but haven't been able to reproduce the failure. It seems to be an issue with the trailing `)`'s indentation, which my setup indents to the same indentation level as the `print` line, but the test suite is indenting to the same indentation as the arguments. 

I have a few settings set on my machine that might be causing issues. The default R language `indentkeys` does not contain `0)`, meaning new lines starting with ) aren't automatically reevaluated for indentation. Using the default `indentkeys`, I believe the expected output should be as follows, but I'm still trying to hunt down where I might have masked this setting to be able to reproduce it. 

```r
print(
  1,
  a = 2,
  3
  )
```

I believe this is an issue with my system settings not reflecting the unit test environment and not an issue with the code. I'm happy to help dig into it more, but it would be very helpful if you wouldn't mind adjusting the expected result to reflect whatever the default indentation produces.

Reply:
The problem has to do with R indentation differences between Vim versions -- the build uses 8.0.1453, which seems to indent the closing brackets like this. I've vendored R support, so that the build has a consistent version.